acceptance/setup/pre_suite/40_install_deps.rb

```diff
@@ -47,7 +47,11 @@
       if test_config[:install_type] == :git then
         case os
         when :debian
-          on database, "apt-get install -y --force-yes openjdk-7-jre-headless rake unzip"
+          on database, "apt-get install -y rake unzip"
```

Comment:
do you need `--force-yes`? also, it looks like now we're only installing openjdk on debian 8? what about other debian variants?